thrift.js

```diff
@@ -82,6 +124,7 @@ function Thrift(options) {
     if (options.allowFilesystemAccess) {
         this.fs = fs;
     }
+    this.asyncLoad = !!(this.fs && this.fs.load);
```

Comment:
If you incorporate my recommendation above about using `fs.readFile` instead of `fs.load`, this gate will not be sufficient to infer whether we want async or sync loading, since the Node.js `fs` module can be used for either. Perhaps we can add `options.asyncLoad` to the options object from within `Thrift.load` before using it to construct the underlying Thrift root module, then all transitively derived Thrift modules.

I think it would be better to infer this from whether you used Thrift.load or new Thrift at the root regardless.

Reply:
I set the default for `asyncReadFile` to `false`, setting it to `true` in case we come from `Thrift.load`.